data/mercator.py

```diff
@@ -132,28 +125,17 @@ def __resample(self, lat_in, lon_in, lat_out, lon_out, var, radius=50000):
             output = self.nc_data.interpolate(input_def, output_def, data.transpose())
 
         if len(origshape) == 4:
-            output = output.reshape(origshape[2:])
+            # un-collapse time and depth axes and
+            # move axes back to original positions.
+            output = output.reshape((
+                origshape[0], # time
+                origshape[1], # depth
+                output.shape[0], # lat
+                output.shape[1] # lon
+            ))
 
         return np.squeeze(output)
 
-    def get_path(self, path, depth, time, variable, numpoints=100, times=None, return_depth=False):
```

Comment:
I've deleted `get_path` from fvcom, nemo, and mercator and moved it into model, since the implementations were identical.